card/mod/05_standard/set/type/search_type.rb

```diff
@@ -217,7 +216,7 @@ def default_search_params
     total = card.count search_params
     return '' if limit >= total # should only happen if limit exactly equals the total
 
-    @paging_path_args = { :limit => limit, :item=> inclusion_defaults[:view] }
+    @paging_path_args = { :limit => limit, :item=> inclusion_defaults(card)[:view] }
```

Comment:
I'm not sure how we want to handle inclusion defaults in this case. Above in line 174 I use the default of every item card but here in the page link we define a default for all items.